yarn.lock

```diff
@@ -2381,6 +2372,15 @@
     "@sentry/utils" "7.54.0"
     tslib "^1.9.3"
 
+"@sentry/core@7.56.0":
```

Comment:
when you see stuff getting duplicated you'll want to do 
```console
npx yarn-deduplicate && yarn install
```

Reply:
Oh I see. Thanks @scttcper.
I imagine this tooling would avoid this to begin with. I'm not used to reviewing lock files in any lang 😅